### DISCUSSION TITLE
Push notifications requirement and Eufy app

### DISCUSSION THREAD

**Discussion by francois-nitenberg** [fuatakgun/eufy_security#316] [Q&A] [answered]
Hi,

I'm new to Eufy. I started with their Doorbell and I was really impressed, so decided to go with them for our alarm system.
I'm a little disappointed to find out that Eufy doesn't allow configuring push notifications at the user level, and as this integration relies on those push notifications to update the entities states, it makes it hard to use Eufy's sensors for anything else than a pure alarm system.

For me, the main benefit of having Eufy entities in HA is that I can reuse the sensors (mainly motion and contact) for automations that are not related to security. Otherwise, it means having many sensors in double!

The only workaround I can imagine is to disable Eufy notifications on our phones (OS level), and then forward the important notifications through the HA companion app, such as intrusion detected whilst armed. Frankly, if that's the only way, then I'll return all my Eufy gear and reimplement my alarm system from scratch with Alarmo. 

Is there any better solution? 
Thanks

**Comment by fuatakgun** [comment, marked as answer]
Sadly not, we need push notifications as of now.

**Reply by omar10wahab** [reply]
What is the actual reason and data that is being disabled when notifications are disabled. I'm just wondering how the app and server is working that even if I turn off just motion detection notifications on the app, how it disables all notifications. 
What needs to happen so that all notifications do not need to be enabled all the time for the integration to work?

**Reply by fuatakgun** [reply]
This add-on is acting as a mobile app, data will either arrive from push notifications almost instantly or will be fetched from eufy cloud servers regularly. Disabling notifications would force to fall back to second and your automations probably wouldn't react as you want.
